MethodOverloading/src/com/javalearning/Main.java

```diff
@@ -41,10 +57,10 @@ public static double calcFeetAndInchesToCentimeters(int inches) {
             return -1;
         }
 
-        int feet = inches / 12;
-        int remainingInches = inches % 12;
-        System.out.println(inches + " inches is equal to " + feet + " feet and " + remainingInches + " inches");
-        return calcFeetAndInchesToCentimeters(feet, remainingInches);
+        //int feet = inches / 12;
+        //int remainingInches = inches % 12;
+        System.out.println(inches + " inches is equal to " + calcFeetToInches(inches) + " feet and " + calcRemainingInches(inches) + " inches");
```

Comment:
in the system.out you are using calcFeetToInches. Instead it should be calcInchesToFeed method call.